cmd/shipperctl/cmd/backup/restore.go

```diff
@@ -157,24 +220,33 @@ func uidOfApplication(shipperClient shipperclientset.Interface, appName, appName
 	return &uid, nil
 }
 
-func updateOwnerRefUid(rel shipper.Release, uid types.UID) (*shipper.Release, error) {
-	ownerReferences := rel.GetOwnerReferences()
-	if n := len(rel.OwnerReferences); n != 1 {
-		key, _ := cache.MetaNamespaceKeyFunc(rel)
-		return nil, fmt.Errorf("expected exactly one owner for Release %q but got %d", key, n)
+func uidOfRelease(shipperClient shipperclientset.Interface, relName, relNamespace string) (*types.UID, error) {
+	release, err := shipperClient.ShipperV1alpha1().Releases(relNamespace).Get(relName, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	uid := release.GetUID()
+	return &uid, nil
```

Comment:
What is the reason you decided to return a `*types.UID` instead of `types.UID`?

Reply:
That's a good question that I have no answer to. This probably shouldn't be a pointer